get_databricks_cluster_eventlog.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Default directory to save results to
+SCRIPT_DIR="$(dirname "$0")"
+results_dir="$SCRIPT_DIR/databricks_cluster_eventlogs"
+mkdir -p $results_dir
+
+function print_usage() {
+  cat <<-EOD
+  Usage: $0 -c <clusterId> [-r <results directory>]
+  Options:
+  -c <cluster ID>  Databricks cluster ID
+  -r <results directory>  directory in which to save cluster eventlog
+  -h                      show this helpful message and exit
+EOD
+}
+
+while getopts c:r:h: flag
```

Comment:
The colon after the h isn't necessary because the option doesn't take an argument.